### PR TITLE
fix(curriculum)missing test in step 27 of city skyline project

### LINKED ISSUE #46478: Test case missing in CSS variable challenge
### Describe the Issue

The site accepts code without the required 
```background-color: var(--building-color4);```
in the .bb4 selector
for Step 27 of the Learn CSS Variables by Building a City Skyline in the New Responsive Web Design course.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-27

### Your code

```
.bb4 {
  width: 11%;
  height: 58%;
}

```

### Expected behavior

Test:
Sorry, your code does not pass. Keep trying.

Hint:
You should add `background-color: var(--building-color4)` in the .bs4 selector.

### Screenshots

_No response_

### System

- Device: PC
- OS: Linux Mint 20
- Browser: Firefox
- Version: 101.0.1 (64-bit)

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46478

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98e3.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ You should give `--building-color4` a value of `#538cc6` in the `:root` selector
 assert.equal(new __helpers.CSSHelp(document).getStyle(':root')?.getPropertyValue('--building-color4').trim(), '#538cc6');
 ```
 
+You should add `background-color: var(--building-color4)` in the `.bb4` selector.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.bb4')?.backgroundColor, 'var(--building-color4)')
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98e3.md:29]:
```suggestion
assert.equal(new __helpers.CSSHelp(document).getStyle('.bb4')?.getPropVal('background-color', true), 'var(--building-color4)')
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

**Comment by ShaunSHamilton** [comment]
@ieahleen Yes please. The `.getPropVal(..., true)` is necessary for CSS functions like `var`, `calc`, etc. Otherwise code like this fails:
```css
 background-color: var( --building-color);
```

**Review by Sboonny — APPROVED** [review]
LGTM

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

Thank you, Ilenia

**Pull request merged**